File: tests/rest-assured-java/src/test/java/com/portfolio/qa/clients/DocsClient.java
package com.portfolio.qa.clients;

import com.portfolio.qa.core.BaseApiClient;
import com.portfolio.qa.core.Endpoints;
import io.restassured.response.Response;

public class DocsClient extends BaseApiClient {

    public Response getDocs() {
        return request()
            .when()
            .get(Endpoints.DOCS);
    }
}

File: tests/cypress-ts/src/clients/docs-client.ts
import { Endpoints } from '../core/endpoints';
import { BaseApiClient } from './base-api-client';

export class DocsClient extends BaseApiClient {
    getDocs(): Cypress.Chainable<Cypress.Response<string>> {
        return this.execute(
            'Get Swagger Documentation',
            'GET',
            Endpoints.docs
        );
    }
}